pynhm/constants.py

```diff
@@ -1,10 +1,19 @@
 import pathlib as pl
+from enum import Enum
 
 import numpy as np
 
 __pynhm_root__ = pl.Path(__file__).parent
 
 zero = np.zeros([1])[0]
 one = np.ones([1])[0]
+nan = np.nan
 
 fill_value_f4 = 9.96921e36
+
+
+class HruType(Enum):
```

Comment:
i noticed you dictched enums on canopy... FWIW

Reply:
@langevin-usgs

Reply:
If I did it was unintentional.  I didn't notice them in there.